### PR TITLE
Feat/custom user agent

### PR DESCRIPTION
**Pull request by rtsun95**
Ability to set a custom userAgent for `damon`

### CODE REVIEW

File: src/bots/worker.js [hunk 1]
```diff
@@ -78,6 +76,9 @@ var logLevel = config.logLevel !== undefined ?
     config.logLevel : 'none';
 var log = require('./log').config.call(casper, pid, logLevel);
 var logger = require('./logger')(cwd);
+var userAgent = config.userAgent !== undefined ?
+    config.userAgent : 'Mozilla/5.0 (Windows NT 6.1; WOW64) ' +
+    'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.157 Safari/537.36';
```
Comment (yoannmoinet) [inline src/bots/worker.js:80]: why not `var userAgent = config.userAgent || 'Mozilla/5.0...';` ?

**Comment by yoannmoinet** [comment]
:+1: 
just a precision, and we'll merge it.

**Comment by rtsun95** [comment]
Fixed the code logic above^

**Comment by yoannmoinet** [comment]
merged (FINALLY)

**Pull request merged**